crates/solidity/testing/snapshots/cst_output/ContractDefinition/unicode_in_doc_comments/generated/0.4.11-failure.yml

```diff
@@ -9,14 +9,19 @@ Source: >
 Errors: # 1 total
   - >
     Error: Expected ContractKeyword.
-       ╭─[crates/solidity/testing/snapshots/cst_output/ContractDefinition/unicode_in_doc_comments/input.sol:1:1]
+       ╭─[crates/solidity/testing/snapshots/cst_output/ContractDefinition/unicode_in_doc_comments/input.sol:4:1]
        │
```

Comment:
> Note2: I see that the error is one off at the end, probably accounting EOF? But was already the case before, so there's no real change there.

Can you please elaborate? `range.end` should be exclusive.

Reply:
Unrelated to this PR:
Just noticed that our `cst_output` tests don't write `error.range.end` to the snapshot file. Maybe we can add it to make spotting bugs like this easier.

Reply:
It's just a visual thing, might even be a problem of the error reporting tool:

```
     2 │ "\✅"
       │ ──┬───  # <-- this line spans one more character than it should
       │   ╰───── Error occurred here.
 ```

Reply:
double checked, and it looks like it is a bug in `ariadne`, where it is also highlighting the `\n` character at the end.